### DISCUSSION TITLE
Where are the GSI "fix" files

### DISCUSSION THREAD

**Discussion by dkokron** [hafs-community/HAFS#165] [General]
I am testing the HAFS GSI component on my workstation.  I am attempting to run the global_fv3_4denvar_C192 case.  Ii gets to the point where the executable is failing due to missing global_anavinfo.l127.txt.  Where can I download the 'fix' directory that contains this file?
Dan

**Comment by BinLiu-NOAA** [comment]
HAFS does not use the GSI fix files from the GSI submodule, instead the related files can be found under parm/levels and/or parm/analysis/gsi
